examples/hello/publisher/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+// A simple example publisher app. Writes a hello message to a grpc pubsub server.
+// You can run the server provided in this examples section.
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/infobloxopen/atlas-pubsub/examples/hello"
+	pubsubgrpc "github.com/infobloxopen/atlas-pubsub/pubsub/grpc"
+	"google.golang.org/grpc"
+)
+
+var topic = flag.String("topic", hello.DefaultTopicName, "the topic to publish to")
+
+func main() {
+	flag.Parse()
+	conn, err := grpc.Dial(":8080", grpc.WithInsecure())
```

Comment:
Since people tend to take examples as their starting points, they should follow best practices. So, for example, the server address should be passed in on the command line and it should be possible to provide TLS creds.

I won't hold up the PR for this - but please create an issue to update the examples to follow best practices.